src/modules/organisation/api/organisation.uc.spec.ts

```diff
@@ -83,4 +87,51 @@ describe('OrganisationUc', () => {
             );
         });
     });
+
+    describe('findAll', () => {
+        const findOrganisationDto: FindOrganisationDto = {
+            kennung: 'kennung',
+            name: 'name',
+            typ: OrganisationsTyp.SCHULE,
+            offset: 0,
+            limit: 0,
+        };
+
+        it('should find all organizations that match with query param', async () => {
+            const firstOrganisation: OrganisationDo<true> = DoFactory.createOrganisation(true);
+            const secondOrganisation: OrganisationDo<true> = DoFactory.createOrganisation(true);
+
+            organisationServiceMock.findAllOrganizations.mockResolvedValue({
+                total: 2,
+                offset: 0,
+                limit: 0,
+                items: [firstOrganisation, secondOrganisation],
+            });
+
+            const result: Paged<OrganisationResponse> = await organisationUc.findAll(findOrganisationDto);
+
+            expect(result.total).toBe(2);
+            expect(result.items).toHaveLength(2);
+            expect(result.items[0]?.name).toEqual(firstOrganisation.name);
+            expect(result.items[1]?.name).toEqual(secondOrganisation.name);
+            expect(result.items[0]?.kennung).toEqual(firstOrganisation.kennung);
+            expect(result.items[1]?.kennung).toEqual(secondOrganisation.kennung);
+            expect(result.items[0]?.typ).toEqual(firstOrganisation.typ);
+            expect(result.items[1]?.typ).toEqual(secondOrganisation.typ);
```

Comment:
Using something like `expect(...).toStrictEqualt(...)` or other Jest built in methods to avoid messy tests.

Reply:
Using the toStrictEqual method causes the test to fail because OrganizationResponse and OrganizationDo have different fields like (createdAt,...)